spritz/README.md

```diff
@@ -0,0 +1,153 @@
+- for each page, optionally, an "autoplay" animation segment
+
+These segments must be consecutive to one another, as if the user continuously moves to one page to another.
+
+### Code!
+
+After getting your `LottieAnimationView` and your `ViewPager`, just create a `Spritz` object with the following syntax:
+
+```
+Spritz
+    .with(lottieAnimationView)
+    .withDefaultSwipeAnimationDuration(300, TimeUnit.MILLISECONDS)
+    .withDefaultSwipeForwardInterpolator(SWIPE_FORWARD_INTERPOLATOR)
+    .withPages(
+            new SpritzPage.Builder()
+                    .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                    .withSwipeDuration(500, TimeUnit.MILLISECONDS)
+                    .build(),
+            new SpritzPage.Builder()
+                    .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                    .withSwipeDuration(500, TimeUnit.MILLISECONDS)
+                    .withSwipeBackwardsInterpolator(SWIPE_BACKWARDS_INTERPOLATOR)
+                    .build(),
+            new SpritzPage.Builder()
+                    .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                    .build()
+    )
+    .attachTo(viewPager);
+```
+
+## Technical documentation
+
+### Make your own kind of Spritz
+
+Start building your view pager animation by calling:
+
+```
+SpritzPage.with(lottieAnimationView)
+```
+
+This will set the specific `LottieAnimationView` as the target of your animations. The Lottie view must load the animation separately, as it is not 
+in the scope of this library.
+
+After that, you can proceed setting your view pager transitions.
+
+#### `withDefaultSwipeAnimationDuration`
+
+`withDefaultSwipeAnimationDuration(long duration, TimeUnit timeUnit)` allows you to specify how long the swiping animation has to complete if the user 
+stops dragging.
+
+#### `withDefaultSwipeForwardInterpolator`
+
+`withDefaultSwipeForwardInterpolator(TimeInterpolator swipeForwardInterpolator)` sets the interpolator to use for all pages to complete the swipe 
+animation if the user stops dragging forwards at any point (before starting the autoplay animation, if any). If you don't set this, it will default to
+`LinearInterpolator`. You can always override this default for single pages (see [`withSwipeForwardInterpolator`](#withswipeforwardinterpolator)).
+
+#### `withDefaultSwipeBackwardsInterpolator`
+
+`withDefaultSwipeBackwardsInterpolator(TimeInterpolator swipeBackwardsInterpolator)` sets the interpolator to use for all pages to complete the swipe 
+animation if the user stops  dragging backwards at any point. If you don't set this, it will default to `LinearInterpolator`. You can always override 
+this default for single pages (see [`withSwipeBackwardsInterpolator`](#withswipebackwardsinterpolator)).
+
+#### `withPages`
+
+`withPages(SpritzPage... pages)` lets you specify the number of pages to animate through. The library is made to work with the same number of pages as 
```

Comment:
_lets you specify the ~number of~ pages to animate through_

I would also explain better the idea behind the next sentence, something like _The number of pages you pass in must match the number of pages on the `ViewPager` you'll attach `Spritz` to_

Reply:
Done as part of e2a51cdb19e2d3e97f72ab0f246ac572131838bd.